Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM node:18-alpine
+
+WORKDIR /usr/src/app
+
+COPY package*.json ./
+# install dependencies for node-gyp (ihate it)
+RUN apk add python3 make gcc g++
+
+RUN npm install
+COPY . .
+# build and generate prisma client
+RUN npx prisma generate 
```

Comment:
prisma generates itself on `npm i` from a postinstall script, otherwise the typescript ci would always fail